openage/opus/setup.py

```diff
@@ -0,0 +1,14 @@
+# this file should be removed in the future.
+# asdf
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+setup(
+    name = 'opusencoder',
+    ext_modules = cythonize([
+        Extension("opusenc",["opusenc.pyx"],
+                   libraries=["ogg","opus"])
+        ])
+)
+
```

Comment:
Hmm, the `pyext_link_library` seems to be non-ideal, as the variable value would descend the recursion of the directory traversal. But it is the only way currently linking against a custom library. Those libraries need to be found first (see the buildsystem folder and the `libopenage/CMakeLists.txt` on how that is done).

Hardcoding `ogg` and `opus` won't work on all platforms, that's why the locations are "found" first and then inserted there.